### PR TITLE
feat(anchor): change props from `iconVisible` to `hideIcon`

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
  - [x] These changes require a usage change
- [x] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [ ] Updated Storybook documents for component
- [ ] Wrote test codes for composable, utils and etc.
- [ ] Tested with console(if usages are changed) 

### Description
- change props from `iconVisible` to `hideIcon`
- change to <script setup>

### CODE REVIEW

File: src/inputs/anchors/PAnchor.vue [hunk 1]
```diff
@@ -31,95 +37,41 @@
     </router-link>
 </template>
 
-<script lang="ts">
-import { computed, defineComponent, ref } from 'vue';
+<script setup lang="ts">
```
Comment (song2kim) [inline src/inputs/anchors/PAnchor.vue:40]: lol setup script!

**Review by WANZARGEN — COMMENTED** [review]
I think story args must be updated too!

**Review by song2kim — APPROVED** [review]
LGTM

File: src/inputs/anchors/PAnchor.vue [hunk 2]
```diff
@@ -31,95 +37,41 @@
...
-        };
-        const anchorRef = ref<HTMLElement|null>(null);
-        const hasText = computed(() => !!anchorRef.value?.textContent);
-        return {
-            validateTarget,
-            anchorRef,
-            hasText,
-            IconPosition,
-        };
-    },
```
Comment (sulmoJ) [inline src/inputs/anchors/PAnchor.vue:121]: cleeeaaan

**Review by sulmoJ — APPROVED** [review]
LGTM!
Please update the storybook document!

**Pull request merged**